index.html

```diff
@@ -17,7 +17,7 @@
                     <p>Shelmith Githinji</p>                    
                 </div> 
                 <a href="#" class="nav">
-                    <img src="images/Icon - Menu.svg" alt="Menu">
+                    <img src="images/Icon-Menu.svg" alt="Menu">
                 </a>            
             </nav>      
 
```

Comment:
It would be better to wrap the `nav` tag inside a `header` tag, So it will make your code looks more professional and also because it is the header of your page 👍 😃 , Kindly fix it 👍.